consent-ws/src/main/java/org/broadinstitute/consent/http/resources/ConsentsResource.java

```diff
@@ -85,4 +94,55 @@ protected Logger logger() {
         return Logger.getLogger("ConsentsResource");
     }
 
+
+    /**
+     * Given that these end-point isn't mapped in swagger follow this steps:
```

Comment:
Should be: 
`Given that this end-point isn't mapped in swagger follow these steps`